lib/components/viewers/stop-viewer.js

```diff
@@ -285,6 +285,11 @@ class StopViewer extends Component {
           <FormattedMessage id="components.StopViewer.noStopsFound" />
         </div>
       )
+      // If geometries are available (and are not a point) and stop times 
+      // are not, it is a strong indication that the stop is a flex stop.
+      if (stopData?.geometries?.geoJson?.type !== 'Point') {
+        contents = <div><FormattedMessage id="components.StopViewer.flexStop" /></div>
```

Comment:
If the server or connection is slow, the flex text will be displayed until the relevant data has loaded (see New England), so probably split the condition.

Reply:
Good catch thank you! Addressed in f5cbd05d30f14b1b08358dd2ad37c818783a4970